templates/components/featured_card.html

```diff
@@ -7,15 +7,11 @@
   <a href="{{ href }}" tabindex="-1" title="{{ title }}">
     <img src="{{ imageSrc }}" alt="{{ imageAlt }}" width="{{ imageWidth }}" height="{{ imageHeight }}" class="featured-card__image">
   </a>
-  <p class="featured-card__tag featured-card__tag--pick">Our pick</p>
+  <p class="featured-card__tag featured-card__pick">Our pick</p>
```

Comment:
Is this element a `tag` or a `pick`?

Reply:
ah that was a left-over after the refactoring - good spot. I'll remove the tag class.